src/vex.js

```diff
@@ -221,13 +228,58 @@ var vex = {
     // Content
     var contentEl = vexInstance.contentEl = document.createElement('div')
     contentEl.classList.add(baseClassNames.content)
+    contentEl.setAttribute('role', 'dialog')
+    contentEl.setAttribute('aria-hidden', false)
+    contentEl.setAttribute('aria-labelledBy', options.message)
     addClasses(contentEl, options.contentClassName)
     contentEl.appendChild(options.content instanceof window.Node ? options.content : domify(options.content))
     rootEl.appendChild(contentEl)
+ 
+    //Accessibility features
+    //Get initial element
+    var focusedElementBefore = document.activeElement;
+    
+    document.querySelector(options.ariaContentMain).setAttribute('aria-hidden', true)
+    
+    if (options.trapTabKey) {
+      contentEl.onkeydown = function(event) {        
+        var focusableElementsString = "a[href], area[href], input:not([disabled]), select:not([disabled]), textarea, button, iframe, object, embed, *[tabindex], *[contenteditable]"
+        var focusableElements = contentEl.querySelectorAll(focusableElementsString)
+        
+        //Tab key pressed
+        if (event.which == 9) {
+          console.log(focusableElements.length)
```

Comment:
Keep console.log's out of production code.

Reply:
Conole.log's have been removed